site-scrapers/CVS.js

```diff
@@ -28,21 +28,39 @@ module.exports = async function GetAvailableAppointments(browser) {
         `${webData.responsePayloadData.currentTime}-0${offsetMountain}:00`
     );
     return webData.responsePayloadData.data.MA.map((responseLocation) => {
-        let hasAvailability = parseInt(responseLocation.totalAvailable)
-            ? true
-            : false;
-        let totalAvailability = parseInt(responseLocation.totalAvailable);
```

Comment:
...can't we just update this to check if totalAvailable is a number?

Reply:
We could. It seems to me the safe thing to do is to not assume that `totalAvailable` will return (right now it is not present, so `parseInt(undefined)` returns `NaN`, and `NaN` is falsy, so the test works.

I'm just not confident that we won't see `status: Available` with no `totalAvaialble` key, meaning we'll still get a falsy `NaN` when we really want to show availability.

I guess this is a judgement call on which is more robust. Have I convinced you?